src/main/java/com/algosenpai/app/storage/Storage.java

```diff
@@ -16,28 +17,19 @@
      * @param filePath path to file.
      * @return A string containing the exact text in the file, including newlines.
      */
-    public static String loadData(String filePath) {
+    public static String loadData(String filePath) throws FileNotFoundException {
         File file = new File(filePath);
 
-        // If the file does not exist, create it.
         if (!file.isFile()) {
-            try {
-                file.createNewFile();
-            } catch (IOException e) {
-                e.printStackTrace();
-            }
-
-            // Fill it with the default UserStats.
-            UserStats newUserStats = UserStats.getDefaultUserStats();
-            saveData(filePath, newUserStats.toString());
-            return newUserStats.toString();
+            throw new FileNotFoundException();
         }
 
         try {
             return Files.readString(Paths.get(filePath), StandardCharsets.US_ASCII);
         } catch (IOException e) {
             e.printStackTrace();
         }
+
```

Comment:
I would like to apologize for what seems like a blatant attempt to increase my line count at the expense of readability. I hope you understand this was an accident

Reply:
I believe Reposense will not detect this extra line